content/docs/concepts/staking/_index.md

```diff
@@ -230,10 +245,21 @@ See [Bonding](/concepts/staking/#bonding) above.
 
 Stake is unbonded from a validator through an unbonding operation. Unbonding is subject to an [unbonding period](/concepts/staking/#unbondingperiod) during which it remains locked. The unbonding period applies irrespective of whether the nominated validator is a member of the consensus committee or not. 
 
-Unbonding is triggered by a staker submitting an `unbond()` transaction. Unbonding can begin as soon as the unbond transaction request has been finalised. On processing the transaction the bonded stake token moves from `bonded` to the intermediate state of `unbonding`. The stake is still locked during the unbonding period. The unbonding request is captured and tracked in memory and the staking transition is applied at the end of the epoch in which the unbonding period expires: unbonding is applied, reducing the validator’s bonded stake amount and so applying the voting power change to the validator.
+Unbonding is triggered by a staker submitting an `unbond()` transaction. Unbonding can begin as soon as the unbond transaction request has been finalized. On processing the transaction the bonded stake token moves from `bonded` to the intermediate state of `unbonding`. The stake is still locked during the unbonding period. The unbonding request is captured and tracked in memory and the staking transition is applied at the end of the epoch in which the unbonding period expires: unbonding is applied, reducing the validator’s bonded stake amount and so applying the voting power change to the validator.
```

Comment:
> The unbonding request is captured and tracked in memory and the staking transition is applied **at the end of the epoch in which the unbonding period expires**: unbonding is applied, reducing the validator’s bonded stake amount and so applying the voting power change to the validator.

The voting power decreases when `_applyUnbonding` is executed, which is at the end of the epoch in which the unbonding request was issued.

The NTN are then released only at the end of the unbonding period

Reply:
Good catch, thanks.  Corrected and reworded in commit 79e8128.

Reply:
And in the API Reference in commit a7ccaca